src/grandorgue/sound/GOSoundEngine.cpp

```diff
@@ -206,7 +206,7 @@ void GOSoundEngine::ClearSetup() {
 }
 
 void GOSoundEngine::Setup(
-  GODefinitionFile *organ_file, unsigned release_count) {
+  GOOrganController *organ_file, unsigned release_count) {
```

Comment:
In similar places you've changed the naming to something like organ_controller, why not here?

Reply:
Renamed to `organControlled`